FluentUI.Demo/src/main/java/com/microsoft/fluentuidemo/demos/ContextualCommandBarActivity.kt

```diff
@@ -0,0 +1,73 @@
+package com.microsoft.fluentuidemo.demos
+
+import android.os.Bundle
+import android.view.View
+import android.widget.Toast
+import com.microsoft.fluentui.contextualcommandbar.CommandItem
+import com.microsoft.fluentui.contextualcommandbar.CommandItemGroup
+import com.microsoft.fluentui.contextualcommandbar.ContextualCommandBar
+import com.microsoft.fluentui.contextualcommandbar.DefaultCommandItem
+import com.microsoft.fluentuidemo.DemoActivity
+import com.microsoft.fluentuidemo.R
+import kotlinx.android.synthetic.main.activity_contextual_command_bar.contextual_command_bar_default
+import kotlinx.android.synthetic.main.activity_contextual_command_bar.contextual_command_bar_with_dismiss
+
+class ContextualCommandBarActivity : DemoActivity() {
+    override val contentLayoutId: Int
+        get() = R.layout.activity_contextual_command_bar
+
+    override fun onCreate(savedInstanceState: Bundle?) {
+        super.onCreate(savedInstanceState)
+
+        val itemGroups = arrayListOf<CommandItemGroup>()
+
+        itemGroups.add(CommandItemGroup()
+                .addItem(DefaultCommandItem(R.drawable.ic_fluent_add_24_regular, "Add"))
```

Comment:
should the description values also be localized (if we are localizing 'Click' for the toast below)?